pkgs/development/python-modules/aioinflux/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchPypi
+, aiohttp
+, ciso8601
+, pandas
+}:
+
+buildPythonPackage rec {
+  pname = "aioinflux";
+  version = "0.9.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1jy5mcg9wdz546s9wdwsgkxhm2ac4dmphd9vz243db39j1m0a3bj";
+  };
+
+  propagatedBuildInputs = [ aiohttp ciso8601 pandas ];
```

Comment:
`pandas` is one of the [`extras_require`](https://github.com/gusutabopb/aioinflux/blob/master/setup.py#L43-L53). 

One opinion is to not add `extras_require` (#62833).

Reply:
I kept that based on the discussion here: https://github.com/NixOS/nixpkgs/pull/70840#discussion_r344992370, but I don't really care one way or the other.

Reply:
If it is a package you normally want to use just add it.